### PR TITLE
Count scrolling

### PR DESCRIPTION
**Pull request by geenen124**
Add pause/play functionality to the client timeline, with bar to indicate current position. The page scrolls as it plays

### CODE REVIEW

File: elements/timeline-grid.html [hunk 1]
```diff
@@ -2,14 +2,28 @@
 <link rel="import" href="timeline-element.html">
 <link rel="import" href="../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
 
-<dom-module id="timeline-grid">
+<dom-module id="timeline-grid" current-count="{{currentCount}}">
```
Comment (bart-de-jonge) [inline elements/timeline-grid.html]: Is this necessary? The current count is defined in the properties of the timeline-grid. I think polymer takes care of the rest

**Comment by bart-de-jonge** [comment]
1 small comment, other than that it looks fine to me.

File: elements/timeline-grid.html (hunk 1 above)
Comment (geenen124) [inline elements/timeline-grid.html]: This is the binding for the current count property. This allows socket.io to pass the current count as needed, at which point the property is updated and Polymer can trigger its property changed handler.

Comment (martijnjanssen) [inline elements/timeline-grid.html]: This actually does work, this keeps all the variables in the same part of the page.

File: elements/timeline-grid.js [hunk 2]
```diff
@@ -125,6 +133,37 @@ class timelineGrid {
         // Set data to property
         this.timeline_data = newData;
     }
+
+    _countChanged(newCount) {
+        if (newCount > 0) {
+            // eslint-disable-next-line no-undef
+            const timeDiv = Polymer.dom(this.root).querySelector("div.timeline-grid");
+            // eslint-disable-next-line no-undef
+            const scrollLine = Polymer.dom(timeDiv).querySelector("#scrolling-box");
```
Comment (martijnjanssen) [inline elements/timeline-grid.js:142]: A shorter selection is `this.$.scrolling-box`.

**Comment by martijnjanssen** [comment]
One minor comment, other than that, looks good.

File: elements/timeline-grid.js (hunk 2 above)
Comment (geenen124) [inline elements/timeline-grid.js:142]: In this case it doesn't work.

**Pull request merged**